source/material_model/viscoelastic.cc

```diff
@@ -186,6 +184,11 @@ namespace aspect
              MaterialModel::MaterialModelOutputs<dim> &out) const
     {
 
+      // Elastic outputs interface
```

Comment:
As we have ElasticOutputs and ElasticAdditionalOutputs (which is a bit confusing) I would change this comment to "Create the structure for the elastic force terms that are needed to compute the right-hand side of the Stokes system".